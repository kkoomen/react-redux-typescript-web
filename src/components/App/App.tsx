import React, { Component, ReactNode } from 'react';

import { DEBUG } from '../../config/constants';
import Debug from '../Debug';

interface Props {};
interface State {};

class App extends Component<Props, State> {
  state: State = {};

  render(): ReactNode {
    return (
      <div className="App">
        {DEBUG && <Debug />}
      </div>
    );
  }
}

export default App
