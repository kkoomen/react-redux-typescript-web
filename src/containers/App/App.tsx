import React, { Component, ReactNode } from 'react';

import { __DEBUG__ } from '../../config/constants';
import Debug from '../../components/Debug';

interface Props {};
interface State {};

class App extends Component<Props, State> {
  state: State = {};

  render(): ReactNode {
    return (
      <div className="App">
        {__DEBUG__ && <Debug />}
      </div>
    );
  }
}

export default App
