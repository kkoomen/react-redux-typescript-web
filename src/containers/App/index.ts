import { App as AppContainer } from './App';

export const App = AppContainer;
