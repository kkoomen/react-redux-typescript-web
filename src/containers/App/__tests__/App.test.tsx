import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import App from '../';

describe('App', () => {
  it('should match its snapshot', () => {
    const element = shallow(<App />);
    expect(toJson(element)).toMatchSnapshot();
  });
});
