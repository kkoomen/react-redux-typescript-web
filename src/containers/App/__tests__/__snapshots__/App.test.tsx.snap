// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match its snapshot 1`] = `
<div
  className="App"
>
  <Connect(Debug) />
</div>
`;
